woocommerce/woocommerce
# Add focus styles for select in Checkout block

When we moved out select to a native select, the code we had for select focus stopped working, or never worked, I'm not sure, but this PR fixes it.

Closes https://github.com/woocommerce/woocommerce/issues/51231



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Click on select in Checkout block and select a value, you should notice a dark outline.
2. Select another field, notice the outline is gone from the select and into the new input.
3. Switch up the theme to something else, notice there's still a notice.
4. Switch up to dark mode in Checkout by going to edit page of Checkout page, selecting the parent Checkout block, and enabling dark mode.
5. Confirm colors still look correct.



### Screenshots

<img width="512" alt="image" src="https://github.com/user-attachments/assets/e25eff4b-1755-4ebc-a907-919ddb610b18">
<img width="687" alt="image" src="https://github.com/user-attachments/assets/24ad3bdb-2a53-48c1-ada8-ae1f8f01c9e4">
<img width="673" alt="image" src="https://github.com/user-attachments/assets/855fee0d-9288-489f-8980-4cd09df3ee4c">

### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Add focus styles to the Checkout block select field.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
